h/templates/layouts/base.html.jinja2

```diff
@@ -78,7 +78,12 @@
     {% endfor %}
   </head>
   <body class="body">
-    {% block content %}{% endblock %}
+    {% block header %}{% endblock %}
+
+    <main>
+      {% block content %}{% endblock %}
+    </main>
+
```

Comment:
Here is where you would put a `{% block footer %}` and have child templates override it to provide a footer if they want to, and the CSS class to make that footer always stick to the bottom could be here in the base template. But that's for a follow-up PR.